src/map/js/main.js

```diff
@@ -49,19 +49,123 @@ let lazyloadAssets = () => {
   loadCSS(`../vendors/kalendae/build/kalendae.css`);
   loadCSS(`https://js.arcgis.com/3.20/esri/css/esri.css`);
  };
+ 
+const parseTitles = (planetBasemaps, isMonthly) => {
```

Comment:
if you want to keep it *clean*, you could always create a new planet basemap module file that is responsible for fetching and and parsing the options, that way you can export out `getPlanetBasemaps()` and the `main.js` file would be more *clean*. Only a suggestion not required at all.